### ISSUE TITLE
Missing space in validity with time

### ISSUE THREAD

**Issue by manuelmaurer** [sibiantony/ssleuth#82]
If the option "Show GMT/UTC time in validity" is activated, the time is directly concatenated to the date making it hard to read:
![original](https://user-images.githubusercontent.com/1051419/29003855-d8fb6cba-7abe-11e7-8f36-4c6f423b491d.png)

I've patched the output to add a space between date and time:
![patched](https://user-images.githubusercontent.com/1051419/29003870-1133080e-7abf-11e7-9c35-cb3b0ad12773.png)

[patch.txt](https://github.com/sibiantony/ssleuth/files/1203119/patch.txt)

**Comment by sibiantony** [comment]
Thanks for reporting. Will apply the patch. 
However, I'm more curious about the SSL/TLS version in your screenshot. Is it always empty ? Do you see the TLS version after a page reload ?

**Comment by rugk** [comment]
And… if possible: Which website is this?

**Comment by manuelmaurer** [comment]
No, currently everything is displayed as intended. I have no idea how I got it into this state.
The screenshots were taken from the New Issue website here at github (https://github.com/sibiantony/ssleuth/issues/new)

I had portage running in the background rebuilding some packages, maybe this interfered. I'll try to keep an eye on it, but I think it was a unique glitch.
